oaknational/Oak-Web-Application
# Fix/lesq 160/hamburger nav a11y [LESQ-160]

## Description

Music year: 2000

- Wrapped the hamburger menu links section in a `nav` element
- Added `aria-expanded` true/false to the menu container when open/closed respectively
- The menu button is already marked as aria-expanded=false when the menu is closed:
- <img src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/245f435a-692c-4f6a-96e3-0fe0cd174198" width=400>
- @jack-skerrett-oak please verify the `nav` is at the correct level as I had some trouble figuring out where it should go!

## How to test

1. Go to https://deploy-preview-2144--oak-web-application.netlify.thenational.academy
2. Click on the hamburger menu
3. You should see the element is a `nav`

## ACs
